src/Wizard/WizardActions/CancelButton/CancelButton.types.ts

```diff
@@ -0,0 +1,10 @@
+export interface CancelButtonProps {
```

Comment:
Looks like the type definition for each of the 3 buttons is identical and could benefit from a shared type, presumably in the level above in wizard types.

Reply:
I guess 'ActionButtonProps'.